src/internet_identity/src/signature_map.rs

```diff
@@ -1,16 +1,17 @@
 //! Maintains user signatures and expirations.
-use certified_map::{AsHashTree, RbTree};
-use hashtree::{leaf_hash, Hash, HashTree};
+use ic_certified_map::{leaf_hash, AsHashTree, Hash, HashTree, RbTree};
+use std::borrow::Cow;
 use std::collections::BinaryHeap;
 
+#[derive(Default)]
```

Comment:
what is this for?

Reply:
The library version of rb_tree requires Default.